### PR TITLE
Added small refactorings, mostly code styling

### PR DESCRIPTION
**Pull request by larssmit**
Suppress Ruby warnings + use add_dependency in gemspec + removed require rubygems

Fixed merge leftovers

Removed duplicate line

### CODE REVIEW

File: lib/zewo.rb [hunk 1]
```diff
@@ -252,7 +226,7 @@ def uncommited_changes?(repo_name)
       end
 
       def master_branch?(repo_name)
-        name = `cd #{repo_name}; git rev-parse --abbrev-ref HEAD`
+        `cd #{repo_name}; git rev-parse --abbrev-ref HEAD`
       end
```
Comment (paulofaria) [inline lib/zewo.rb:230]: this `name` variable isn't an instance variable right? that's why you removed it, correct?

File: lib/zewo.rb [hunk 2]
```diff
@@ -341,8 +310,13 @@ def checkout()
...
         if options[:branch]
           matched = options[:branch]
+          matched = `cd #{repo.name} && git tag`
+                    .split("\n")
+                    .select { |t| t.start_with?(options[:tag]) }
+                    .last
         end
 
+        matched = options[:branch] if options[:branch]
         if matched
```
Comment (paulofaria) [inline lib/zewo.rb:308]: I don't think I understand this part. why do you assign `matched = options[:branch]` and then later

```
matcher = `cd #{repo.name} && git tag`
                     .split("\n")
                     .select { |t| t.start_with?(options[:tag]) }
                     .last
```

 this makes `matched = options[:branch]` kinda useless doesn't it?. and later you do `matched = options[:branch] if options[:branch]` which makes the whole code before useless too, right? am I crazy? haha

Also I sent you a DM in Zewo's slack. You should check later! (:
Again, thanks a lot for this PR.

File: lib/zewo.rb (hunk 1 above)
Comment (larssmit) [inline lib/zewo.rb:230]: Indeed, and you never use the name variable in this method. Besides that, Ruby returns always the last evaluated expression so you don't have to explicitly use 'return'.

File: lib/zewo.rb (hunk 2 above)
Comment (larssmit) [inline lib/zewo.rb:308]: You're not crazy, I am ;-). I fixed this in a later commit.

**Comment by larssmit** [comment]
Do I need to close this merge request and make a new one to squash it to one commit? (Github has the ability to do pull requests merge squashing (if you configured this): https://help.github.com/articles/about-pull-request-merge-squashing/)

**Comment by paulofaria** [comment]
we can squash ourselves! LGTM @Danappelxx ?

**Comment by Danappelxx** [comment]
Yeah! LGTM @paulofaria can you merge, squash, and tag please?

**Pull request merged**